meta/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
 
 dependencies:
+  - role: debops.apt_preferences
+    tags: [ 'depend::apt_preferences', 'type::dependency', 'depend-of::gitlab', 'type::dependency' ]
+    apt_preferences_dependent_list:
+      - package: 'golang*'
```

Comment:
This should probably be `'golang golang-*` to be more explicit about the packages you select.